app/views/tags/edit.html.slim

```diff
@@ -0,0 +1,7 @@
+h1 Editing tag
```

Comment:
i guess this was autogenerated. is it in use? If yes, do we have to use locales as well?